articles/azure/azs-how-to-retrieve-billing-api.md

```diff
@@ -0,0 +1,216 @@
+---
+title: How to retrieve Billing information from the CSP Portal via API
+description: How to retrieve Billing information from the CSP Portal via API
+services: azure-stack
+author: David Woffendin
+
+toc_rootlink: Operators
+toc_sub1: How To
+toc_sub2: 
+toc_sub3:
+toc_sub4:
+toc_title: Retrieve Billing information from the CSP Portal via API
+toc_fullpath: Operators/How To/azs-how-to-retrieve-billing-api.md
+toc_mdlink: azs-how-to-retrieve-billing-api.md
+---
+# Retrieving Billing information from the CSP Portal via API
+The purpose of this guide is to help obtain Billing information from the CSP Portal via the API.
+
+## Authentication 
```

Comment:
We usually include an empty line after each heading. This is a requirement of the markdown, but I don't think it actually affects how the article is rendered. I do think it also makes the files easier to read though. This will need changing throughout the article.

Reply:
amended throughout document